src/kvstore/raftex/RaftPart.cpp

```diff
@@ -567,16 +597,14 @@ void RaftPart::replicateLogs(folly::EventBase* eb,
             VLOG(2) << self->idStr_
                     << "Appending logs to "
                     << hostPtr->idStr();
-            return via(
-                eb,
-                [=] () -> Future<cpp2::AppendLogResponse> {
-                    return hostPtr->appendLogs(eb,
-                                               currTerm,
-                                               lastLogId,
-                                               committedId,
-                                               prevLogTerm,
-                                               prevLogId);
-                });
+            return via(eb, [=] () -> Future<cpp2::AppendLogResponse> {
```

Comment:
I think we can write more "c++" style code to replace "gen::from|gen::map|gen::as", because it hard to debug in gdb.